spec/data-model/README.md

```diff
@@ -214,11 +217,43 @@ that the implementation attaches to that _annotation_.
 ```ts
 interface UnsupportedAnnotation {
   type: "unsupported-annotation";
-  sigil: "!" | "@" | "#" | "%" | "^" | "&" | "*" | "<" | ">" | "/" | "?" | "~";
+  sigil: "!" | "@" | "%" | "^" | "&" | "*" | "+" | "<" | ">" | "?" | "~";
   source: string;
 }
 ```
 
+## Markup
+
+Each of _markup-open_, _markup-close_, and _markup-standalone_ are represented
+by the corresponding `MarkupOpen`, `MarkupStandalone`, and `MarkupClose`.
```

Comment:
This reads oddly (both in terms of English and in terms of stylistically). The other sections start with the interface names, while this starts with what is represented. Perhaps:

```suggestion
Each of the markup interfaces `MarkupOpen`, `MarkupStandalone`,
and `MarkupClose` represent the _markup_ `type` of the corresponding
`kind`.
```